### ISSUE TITLE
Rasperry Pi Pico as USB-to-MIDI converter

### ISSUE THREAD

**Issue by renne** [alf45tar/PedalinoMini#286]
@alf45tar 
The [Raspberry Pi Pico](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) costs a third of the Arduino Pro Micro and is much better available.

Please add instructions how to use a Raspberry Pi Pico as an USB-to-MIDI converter instead of an Arduino Pro Micro.

**Comment by touchgadget** [comment]
I have not tried PedalinoMini yet but there are other USB MIDI to/from DIN programs for 3.3V uC such as samd21 and Pi Pico (RP2040). SparkFun and Adafruit in the US sell Raspberry Pi Pico boards for $4.

https://github.com/gdsports/MIDIUARTUSB

**Comment by MiqViq** [comment]
I am planning to test some small boards with SAMD21 or RP2040 as MIDI-USB-handler, as soon I get my shipments from China. I will report my findings here...

**Comment by Sthopeless** [comment]
> In Germany you can get the Raspberry Pi Pico easily for 4,- € in most electronics stores. The Arduino Pro Micro is about 12,- € on Amazon Marketplace Germany (chinese vendors) or 9,- € on eBay. The Raspberry Pi Pico firmware can be stored via generic USB mass storage controller while the Arduino Pro Micro needs device-specific flashing tools for serial-over-USB.

Arduino Pro Micros cost around 5 euros in Aliexpress my original Raspberry Pico from Amazon was around 8 euros (I am aware I can order cheaper from Aliexpress), some STM32's are also capable and less expensive. 

Anyways I managed to get USB working on PedalinoMini using a Pico with this: https://github.com/rsta2/pico
I simply flashed the uf2 file provided here: https://github.com/rsta2/pico/releases/tag/v1.0

Connected my PedalinoMini RX to Pico GPIO0 and PedalinoMini TX to Pico GPIO1 and ground to ground.
I was also able to power my PedalinoMini board just feeding the power from the VBUS port on the Pico.
